control/trajectory_follower_node/src/controller_node.cpp

```diff
@@ -231,7 +233,10 @@ void Controller::callbackTimerControl()
   out.longitudinal = lon_out.control_cmd;
   control_cmd_pub_->publish(out);
 
-  // 6. publish debug marker
+  // 6. publish published time only if there are subscribers more than 1
+  published_time_publisher_->publish(control_cmd_pub_, out.stamp);
+
+  // 7. publish debug marker
```

Comment:
```suggestion
```
We should remove unnecessary comment lines.

Reply:
Sorry, I couldn't understand what is unnecessary here. The comments are ordered in code, I just added another step here:

```
  // 6. publish published time only if there are subscribers more than 1
  published_time_publisher_->publish(control_cmd_pub_, out.stamp);

  // 7. publish debug marker
  publishDebugMarker(*input_data, lat_out);
```